script/create-dist.py

```diff
@@ -152,6 +152,10 @@ def create_symbols():
     dsyms = glob.glob(os.path.join(OUT_DIR, '*.dSYM'))
     for dsym in dsyms:
       shutil.copytree(dsym, os.path.join(DIST_DIR, os.path.basename(dsym)))
+  elif PLATFORM == 'win32':
+    pdbs = glob.glob(os.path.join(OUT_DIR, '*.pdb'))
```

Comment:
I was trying this branch locally and the generated pdb zip was empty, looks like `OUT_DIR` is `out/R` and the `.pdb` files are in `out/D`, am I doing something wrong?